djgeojson/fields.py

```diff
@@ -44,7 +43,12 @@ def __call__(self, value):
 
 
 class GeoJSONFormField(JSONFormField):
-    widget = LeafletWidget if HAS_LEAFLET else HiddenInput
+    widget = None
+    if HAS_LEAFLET:
+        widget = LeafletWidget
+    else:
+        widget = HiddenInput
+        warnings.warn('`django-leaflet` is not available.')
```

Comment:
Wao! I wouldn't have thought that raising a warning within a class definition would even work :)